content/docs/services/cloud-gov-identity-provider.md

```diff
@@ -4,7 +4,7 @@ menu:
     parent: services
 title: cloud.gov identity provider
 name: "cloud-gov-identity-provider"
-description: "Manage client credentials for authenticating cloud.gov users in your app"
+description: "Authenticate cloud.gov users in your app"
```

Comment:
> Authenticate cloud.gov users in your app

This came from me. The thing that stuck out for me with the "Manage client credentials for authenticating cloud.gov users in your app" copy was the "manage" part. The service does no managing of credentials. It only provides you with client id/secret so that you can "Use cloud.gov authentication in your application as a login system for your users". Is that right @jmcarp?

Anyway, I'm happy to go ahead with all the suggested changes above. I don't mean to steamroll any previous thought and discussion for the identity-provider copy, but I would like another round to clarify the thinking behind it and maybe iterate on the existing text.